cytotable/utils.py

```diff
@@ -99,6 +99,24 @@ def Parsl_AppBase_init_for_docs(self, func, *args, **kwargs):
 AppBase.__init__ = Parsl_AppBase_init_for_docs
 
 
+def _parsl_loaded() -> bool:
+    """
+    Checks whether Parsl configuration has already been loaded.
+    """
+
+    try:
+        # try to reference Parsl dataflowkernel
+        parsl.dfk()
+    except RuntimeError as rte:
+        # if we detect a runtime error that states we need to load config
+        # return false to indicate parsl config has not yet been loaded.
+        if str(rte) == "Must first load config":
+            return False
+
+    # otherwise we indicate parsl config has already been loaded
```

Comment:
Just confirming that this is designed to only catch one kind of runtime error and all others will be handled differently?

Reply:
I think this is catching this error:

> parsl/dataflow/dflow.py:            raise RuntimeError('Must first load config')

which I think parsl should probably make its own error type to support this kind of detection without looking at the error text.

I made an issue https://github.com/Parsl/parsl/issues/2873

Reply:
Thanks @gwaybio ! Yes, this is only catching Parsl configuration load exceptions as an indication of whether configurations have already been loaded. I'm adding a `raise` here for other `RuntimeExceptions`.

Thanks as well @benclifford ! I've added an issue to help make the updates here once there is a custom Exception: https://github.com/cytomining/CytoTable/issues/94 .